src/sentry/api/event_search.py

```diff
@@ -140,6 +140,7 @@ def translate(pat):
 quoted_key           = ~r"\"([a-zA-Z0-9_\.:-]+)\""
 
 date_format          = ~r"\d{4}-\d{2}-\d{2}(T\d{2}:\d{2}:\d{2}(\.\d{1,6})?)?Z?(?=\s|$)"
+alt_date_format      = ~r"\d{4}-\d{2}-\d{2}(T\d{2}:\d{2}:\d{2}(\+\d{2}:\d{2})?)?(?=\s|$)"
```

Comment:
Could this format also have microseconds in it?

Reply:
Hmm yeah according to the spec it can. I'll add that.